### DISCUSSION TITLE
打勾选项

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#14181] [Q&A]
![861612462](https://github.com/user-attachments/assets/06cbc155-4451-4914-8c16-faf638c729

有什么好办法能够检测图中红色表格每项的勾，并且能找到对应A，B,C哪一项？

**Comment by cuicheng01** [comment]
表格识别+后处理就可以

**Reply by cqray1990** [reply]
> 表格识别+后处理就可以 
> 手写的打勾后处理如何更好判断单元格有勾呢，用模板匹配匹配不大，每个手写打勾有点差异
